### ISSUE TITLE
[MU4 Issue] Accessibility: missing screenreader support for "Add palettes" menu

### ISSUE THREAD

**Issue by MarcSabatella** [musescore/MuseScore#16067]
**Describe the bug**
When using a screenreader, nothing is read for the "Add palettes" menu

**To Reproduce**
1. Start NVDA or Narrator
2. Click "Add Palettes" button in palettes window
3. Hear error: screenreader does not read anything

**Expected behavior**
The buttons should read the names of the palettes to be added

**Platform information**
 - OS: Windows

**Comment by oktophonie** [comment]
Closing in favour of the duplicate https://github.com/musescore/MuseScore/issues/16744
